src/integrations/mod.rs

```diff
@@ -50,19 +50,29 @@ chrome:
   window_size: [1024, 768]
   idle_browser_timeout: 600
 
+
 # Postgres database
 #postgres:
 #  psql_binary_path: "/path/to/psql"  # Uncomment to set a custom path for the psql binary, defaults to "psql"
 #  connection_string: "postgresql://username:password@localhost/dbname"  # To get a connection string, check out https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING
 
-// cmdline:  # things you can call and immediately get an answer (blocking command line calls)
-//   run_make:
-//     command: "make"
-//     command_workdir: "%project_path%"
-//     timeout: 600
-//     description: "Run `make` inside a C/C++ project, or a similar project with a Makefile."
-//     parameters:
-//       - name: "project_path"
-//         description: "absolute path to the project"
+
+# Command line: things you can call and immediately get an answer
+#cmdline:
+#  run_make:
+#    command: "make"
+#    command_workdir: "%project_path%"
```

Comment:
why didn't you leave a template structure "make %project_path%"?

Reply:
because this is how you call `make`
try to recall -C without googling

Reply:
this structure is not abstract enough that I would be able to execute any command